diff --git a/depends/work/build/x86_64-pc-linux-gnu/qt/5.9.8-3ee65b34332/qtbase/src/corelib/tools/qbytearraymatcher.h b/depends/work/build/x86_64-pc-linux-gnu/qt/5.9.8-3ee65b34332/qtbase/src/corelib/tools/qbytearraymatcher.h
index c1c0c3a66..1e846024b 100644
--- a/qtbase/src/corelib/tools/qbytearraymatcher.h
+++ b/qtbase/src/corelib/tools/qbytearraymatcher.h
@@ -41,6 +41,7 @@
 #define QBYTEARRAYMATCHER_H
 
 #include <QtCore/qbytearray.h>
+#include <iostream>
 
 QT_BEGIN_NAMESPACE
 
